cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -156,7 +159,14 @@ public void setVoltageAngleNodeBreaker() {
         if (!checkValidVoltageAngle(null)) {
             return;
         }
-        VoltageLevel vl = voltageLevel();
+        VoltageLevel vl;
+        if (insideBoundary() && context.config().convertBoundary()) {
```

Comment:
Maybe this logic for getting the right voltage level should go in the `voltageLevel()` function

Reply:
Ok, done in the next commit